packages/core/content-type-builder/admin/src/components/ComponentCard/index.js

```diff
@@ -7,15 +7,77 @@
 import React from 'react';
 import { get } from 'lodash';
 import PropTypes from 'prop-types';
+import { Text, Stack } from '@strapi/parts';
+import { pxToRem } from '@strapi/helper-plugin';
+import { Close } from '@strapi/icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styled from 'styled-components';
 
 import useDataManager from '../../hooks/useDataManager';
-import Wrapper from './Wrapper';
-import Close from './Close';
 
 /* eslint-disable jsx-a11y/click-events-have-key-events */
 /* eslint-disable jsx-a11y/no-static-element-interactions */
 
+const StyledFontAwesomeIcon = styled(FontAwesomeIcon)`
+  width: ${pxToRem(32)} !important;
+  height: ${pxToRem(32)} !important;
+  padding: ${pxToRem(9)};
+  border-radius: ${pxToRem(64)};
+  background: ${({ theme }) => theme.colors.neutral150};
+  path {
+    fill: ${({ theme }) => theme.colors.neutral500};
+  }
+`;
+
+const CloseButton = styled.div`
```

Comment:
I think this should be a button.

Reply:
You can't nest a button inside a button, do a create a div with the role and behaviour of a button

Reply:
> You can't nest a button inside a button, do a create a div with the role and behaviour of a button

A role button is the same for ATs as a regular button IIRC, I'm not 100% it's the way to go